src/components/utils/CMakeLists.txt

```diff
@@ -110,8 +110,20 @@ else ()
     add_subdirectory(./src/sqlite_wrapper)
 endif ()
 
-if(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
- list(APPEND LIBRARIES dl pthread ${RTLIB})
+if(UNIX AND NOT QNXNTO)
```

Comment:
@BSolonenko confusing setting libraries in IF, and sources in ELSE. Please do some small refactor to avoid it

Reply:
@AKalinich-Luxoft Fixed in b2b51c312fc57007af9a6626c0e88bfe2f0b594e